src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1935,6 +1945,12 @@ public static PartitionIterator read(SinglePartitionReadCommand.Group group,
                                                                                       group.metadata(),
                                                                                       group.queries,
                                                                                       consistencyLevel);
+        // Request sensors are utilized to track usages from all replicas serving a read request
```

Comment:
Technically we don't need `requestSensors` on the coordinator but:
* It spares polluting code paths between when responses are received and when `ResultMessage` is constructed. This is more evident on the write code path where lots of methods in between are essentially `void`
* If we end up calculating rates on the coordinator, it enables the usage of `SensorsRegistry` which will come in handy